lib/src/config.dart

```diff
@@ -68,6 +74,12 @@ class TestHtmlBuilderConfig {
 
   final bool browserAggregation;
 
+  final bool randomizeAggregation;
+
+  // This is meant to mirror what's happening in the dev test package if you see how the default value is determined
+  // https://github.com/dart-lang/test/blob/c586cff0f415f9c3006175352b9634ba900fd7d2/pkgs/test_core/lib/src/runner/configuration/args.dart#L259-L261
+  final num testShuffleSeed;
```

Comment:
Instead of having two different options for this, I think just having one for the seed is sufficient (same thing the test runner does).

- If it's not configured, then randomization is disabled.
- If it's set to `random`, then we pick a seed at random during the build.
- Otherwise, we assume that it's set to a num and use that value as the seed.

Additionally, to match the test runner's flag of `--test-randomize-ordering-seed`, I think we should name the field `randomizeOrderingSeed`.